scripts/ci/sync_extensions.py

```diff
@@ -15,14 +15,15 @@
 STORAGE_ACCOUNT = os.getenv('AZURE_EXTENSION_TARGET_STORAGE_ACCOUNT')
 STORAGE_CONTAINER = os.getenv('AZURE_EXTENSION_TARGET_STORAGE_CONTAINER')
 COMMIT_NUM = os.getenv('AZURE_EXTENSION_COMMIT_NUM') or 1
+BLOB_PREFIX = os.getenv('AZURE_EXTENSION_BLOB_PREFIX')
```

Comment:
What will happen if it is not set?

Reply:
It will be `None` and the blob will be uploaded without a prefix in the container.